packages/react-charts/src/components/Skeletons/examples/skeletons.md

```diff
@@ -113,6 +113,7 @@ export const ChartAreaSkeleton: React.FunctionComponent = () => {
     </>
   );
 }
+ChartAreaSkeleton.displayName = 'ChartAreaSkeleton';
```

Comment:
Why is `displayName` necessary here? I don't see this in any other chart example

Reply:
The skeleton examples do not render at all. See https://patternfly-react-pr-10606.surge.sh/charts/skeletons

![Screenshot 2024-06-13 at 7 53 30 AM](https://github.com/patternfly/patternfly-react/assets/17481322/e9bab5ce-f264-4cd9-be24-92f8f328af0d)

Reply:
This was probably inserted by the ESLint plugin for PatternFly, it has a bug that makes it insert this repeatedly. I'll take a look if I can remove this.